conf/kubernetes-resource-staging-server.yaml

```diff
@@ -1,6 +1,6 @@
 ---
-apiVersion: extensions/v1beta1
-kind: ReplicaSet
+apiVersion: apps/v1beta1
+kind: Deployment
```

Comment:
Hmm.. maybe we should use extensions.v1beta1/Deployment instead of the apps group here. The apps group move happened in 1.6 (https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG.md#deployment-2). I'll verify with the team about the plan for this moving forward.